alcs-frontend/src/app/features/home/gis/gis-subtasks.component.html

```diff
@@ -20,8 +20,8 @@ <h4>GIS Subtasks: {{ subtasks.length }}</h4>
           <span class="high-priority" *ngIf="subtask.application.card.highPriority">!</span>
         </td>
         <td>{{ subtask.application.fileNumber }} ({{ subtask.application.applicant }})</td>
-        <td>{{ subtask.application.activeDays }}</td>
-        <td [innerHTML]="subtask.application.status"></td>
+        <td>{{ subtask.reconsideration ? '-' : subtask.application.activeDays }}</td>
+        <td>{{ subtask.application.status }}</td>
```

Comment:
Looks like due to conflicts you grabbed the wrong one, it should be the innerHtml version